src/daft-catalog/src/errors.rs

```diff
@@ -1,5 +1,14 @@
 use snafu::Snafu;
 
+pub type Result<T, E = Error> = std::result::Result<T, E>;
+
+#[macro_export]
+macro_rules! unsupported {
```

Comment:
we've been using `_err` suffix for macros that return the err
```suggestion
macro_rules! unsupported_err {
```